From: Rudy Godoy <rudy@stone-head.org>
Date: Tue, 15 May 2012 01:48:04 -0500
Subject: Added LT_INIT to configure.in.

---
 configure.in |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/configure.in b/configure.in
index be4c7d4..585989c 100644
--- a/configure.in
+++ b/configure.in
@@ -15,8 +15,10 @@
 #
 ##############################################################################
 
+LT_INIT
 AC_INIT(Make-config.in)
 AC_CONFIG_HEADERS(config.h)
+AC_CONFIG_MACRODIR([m4])
 AM_INIT_AUTOMAKE(torcs, 1.3.3)
 
 dnl Checks for programs.
